examples/run_wasm/src/main.rs

```diff
@@ -1,3 +1,21 @@
+/// Use [cargo-run-wasm](https://github.com/rukai/cargo-run-wasm) to build an example for web
+///
+/// Usage:
+/// ```
+/// cargo run_wasm --package [example_name]
+/// ```
+/// Generally:
+/// ```
+/// cargo run_wasm --package with_winit
+/// ```
+
 fn main() {
+    // HACK: We rely heavily on compute shaders; which means we need WebGPU to be supported
```

Comment:
I'm not going to block the PR on this, but I'd word the comment differently. It enables an unstable web feature, and it's basically a TODO to remove the flag when it gets stabilized and we bump the dependency on web_sys to a version that includes WebGPU. That's not exactly a hack, at least in my opinion.